runtime/packages/modules/balances/assembly/balances-module.ts

```diff
@@ -57,6 +57,8 @@ export class BalancesModule {
         this.setBalance(receiver, receiverNewBalance, receiverAccData.getReserved());
         System.incAccountNonce(sender);
         Log.info("done transfering: " + amount.toString());
+        Log.info(senderAccData.getFree().toString() + " - " + senderNewBalance.toString());
+        Log.info(receiverAccData.getFree().toString() + " - " + receiverNewBalance.toString());
```

Comment:
I am not sure that we should log any of this. If we are logging it, we should properly format it. IMO it would be confusing if someone sees the following log:

"{old sender balance} - {new sender balance}"
"{old receiver balance} - {new receiver balance}"
- There is no indication of whose balances are changed and what does "-" mean, is it subtraction?